pkgs/development/libraries/wxwidgets/3.1/default.nix

```diff
@@ -0,0 +1,15 @@
+{ stdenv, fetchFromGitHub
+, common
+}:
+
+common.overrideDerivation (super : rec {
```

Comment:
not sure about using overrideDerivation in of nixpkgs

Reply:
Yeah... on second thought the override should be in top-level like this:
```
wxGTK31 = wxGTK30.overrideDerivation (super : rec {
...
)};
```
Or are you saying override should be avoided entirely? It seems to be used a handful of times in top-level.